### PR TITLE
Feat/detail page

### PR DESCRIPTION
**Pull request by HenrikeStahlhut**
Plants now have a detail page where the user can edit the plant 🕺

### CODE REVIEW

File: pages/plants/[id].js [hunk 1]
```diff
@@ -46,16 +75,41 @@ export default function PlantDetailPage() {
     return <StyledLoading>Loading your rooms...</StyledLoading>;
   }
 
+  console.log(plantTypeName(plant));
```
Comment (JohannSiemens) [inline pages/plants/[id].js:78]: If this console.log is unnecessary you might want to delete it.

**Review by JohannSiemens — COMMENTED** [review]
Good job, Henrike. Just two minor comments from my side. I am not sure how the coaches value this, but i would change the if-else-block into a switch, because it is generally more efficient and easier to read, when you have 5+ statements, but what you did also works.

File: utils/PlantTypeName.js
[the remark this answers is not in the document]
Comment (HenrikeStahlhut) [reply utils/PlantTypeName.js:1]: @JohannSiemens Thats such a good idea, thanks! I changed the if else statements to switch, i didn't think about that

**Review by doctype-Melvin — COMMENTED** [review]
Nicely done Henny!
LGTM

**Pull request merged**